src/MetaMorpho.sol

```diff
@@ -396,10 +404,21 @@ contract MetaMorpho is ERC4626, ERC20Permit, Ownable2Step, Multicall, IMetaMorph
         }
     }
 
-    /* ONLY GUARDIAN FUNCTIONS */
+    /* REVOKE FUNCTIONS */
+
+    /// @notice Revokes the pending cap of the market defined by `id`.
+    function revokeFee() external onlyOwner {
```

Comment:
The `onlyOwner` modifier does not make sense. The `owner` itself is the one that submit the new fees via `submitFee`, why would it be the one being able to revoke the pending change? 

Should it be the guardian, the one able to revoke such pending change?